crates/agglayer-types/src/lib.rs

```diff
@@ -39,6 +40,7 @@ pub struct CertificateHeader {
     pub certificate_id: CertificateId,
     pub new_local_exit_root: Hash,
     pub status: CertificateStatus,
+    pub metadata: Metadata,
```

Comment:
Can we place the `metadata` before `status` so that we have consistent size until the `status` which can contains different length?